src/results/outages.jl

```diff
@@ -67,13 +74,24 @@ function add_outage_results(m, p, d::Dict)
 	r["unserved_load_per_outage_kwh"] = round.(unserved_load_per_outage, digits=2)
 	r["storage_microgrid_upgrade_cost"] = value(m[:dvMGStorageUpgradeCost])
 	r["microgrid_upgrade_capital_cost"] = r["storage_microgrid_upgrade_cost"]
+    _n = ""
 	if !isempty(p.s.storage.types.elec) && Bool(round(value(m[:binMGStorageUsed]), digits=0))
 		r["storage_discharge_series_kw"] = value.(m[:dvMGDischargeFromStorage]).data
+        electric_storage_energy_capacity_kwh = round(value(m[Symbol("dvStorageEnergy"*_n)]["ElectricStorage"]), digits=2)
+        r["soc_series_fraction"] = round.(value.(m[:dvMGStoredEnergy][:,:,1:end]).data ./ electric_storage_energy_capacity_kwh, digits=3)
 	else
-		r["storage_discharge_series_kw"] = []
+		r["storage_discharge_series_kw"] = value.(m[:dvMGDischargeFromStorage]).data
```

Comment:
Should this be = []? The same line is in the if part above, so if not then move out of the if else?
Also should soc_series_fraction = [] on the next line instead of zeros array? I think that's generally what we do when a tech isn't included right?

Reply:
Yep, addressed in 6e3e85a75d1a6f6ce5f6d3df40b10801e20beaed.